So I notice that you have updated the `_general_utils.py` 1 hour ago, and corrected your `gradle_path` variable yourself!
But another issue comes up now:
```
WARNING:[APK Builder]: *** NOTICE: build.gradle file has the line: "signingConfig signingConfigs.release". If you want to use KeyStoreProperties, remove this line from your build.gradle
INFO:[APK Builder]: Preparing build.gradle file for automated release...
INFO:[APK Builder]: Clearing latest .apk file from project...
INFO:[APK Builder]: Building the apk...
'.' is not recognized as an internal or external command,
operable program or batch file.
Traceback (most recent call last):
  File "C:\Users\Ali\Desktop\testapkbuilder\bot.py", line 20, in <module>
    apk_builder.build_apk(project_path='C:/Users/Ali/Desktop/BCSF17M555_TermProject/SuperAnimeDownloader_android',
  File "C:\Users\Ali\AppData\Local\Programs\Python\Python39\lib\site-packages\os_android_apk_builder\apk_builder.py", line 40, in build_apk
    return _build(project_path, apk_dst_dir_path, key_store_properties, version_properties, gradle_path, _res.APK, remove_older_apks_from_output_dir)
  File "C:\Users\Ali\AppData\Local\Programs\Python\Python39\lib\site-packages\os_android_apk_builder\apk_builder.py", line 102, in _build
    raise Exception(f"ERROR: Failed to create {binary_type} file")
Exception: ERROR: Failed to create apk file
```
Here the gradle command becomes `./gradlew assembleRelease`, and `./` causes error on cmd:
```
'.' is not recognized as an internal or external command,
operable program or batch file.
```

### The code I used according to your `README`:
```
from os_android_apk_builder.objs.KeyStoreProperties import KeyStoreProperties
from os_android_app_version_changer.objs.VersionProperties import VersionProperties


# set your KeyStore properties
key_store_props = KeyStoreProperties(key_store_file_path='C:/Users/Ali/Desktop/sample-keystore.jks',
                                     store_password='abcd1234',
                                     key_alias='sample-alias',
                                     key_password='abcd1234',
                                     v1_signing_enabled=True,
                                     v2_signing_enabled=True)

# set the version properties (version code and version name)
version_props = VersionProperties(new_version_code=VersionProperties.RAISE_VERSION_BY_ONE,
new_version_name="1.0.3")


from os_android_apk_builder import apk_builder

apk_builder.build_apk(project_path='C:/Users/Ali/Desktop/BCSF17M555_TermProject/SuperAnimeDownloader_android',
                      apk_dst_dir_path='C:/Users/Ali/Desktop/BCSF17M555_TermProject',
                      key_store_properties=key_store_props,
                      version_properties=version_props)
```

--- a/os_android_apk_builder/bp/_general_utils.py
+++ b/os_android_apk_builder/bp/_general_utils.py
@@ -14,4 +14,3 @@ def release_binary(release_command, project_path, gradle_path):
     gradle_command = f'{gradle_path} {release_command}'
     full_command = f'{cd_command} && {gradle_command}'
     os.system(full_command)
-    print(gradle_command)

--- a/os_android_apk_builder/bp/_general_utils.py
+++ b/os_android_apk_builder/bp/_general_utils.py
@@ -1,20 +1,17 @@
 # system
 import os
 
-##########################################
-# just a boilerPlate for the apk release #
-##########################################
+
+#####################################################
+# just a boilerPlate for the apk/app bundle release #
+#####################################################
 
 
 # will run the command which will create the binary (apk/app bundle)
 def release_binary(release_command, project_path, gradle_path):
-
-    # if the gradle path is None, we will use the project's gradle wrapper
-    if gradle_path is None:
-        gradle_path = os.path.join(project_path, 'gradlew')
-
     cd_command = f'cd {project_path}'
-    #gradle_command = f'{gradle_path} {release_command}'
-    gradle_command = f'gradlew {release_command}'
+    gradle_path = '.\\gradlew' if gradle_path == './gradlew' and os.name == 'nt' else gradle_path
+    gradle_command = f'{gradle_path} {release_command}'
     full_command = f'{cd_command} && {gradle_command}'
     os.system(full_command)
+    print(gradle_command)

